_data/menu_docs_dev.json

```diff
@@ -9,10 +9,6 @@
       "page": "Documentation",
       "slug": "",
       "mainfolderitems": [
-        {
-          "page": "Overview",
-          "url": "index"
-        },
```

Comment:
As mentioned, this page is pointed to both the the docs and the sitemap link. I had to choose a single location in order to get a unique navigation order.